### DISCUSSION TITLE
Cannot change override type.

### DISCUSSION THREAD

**Discussion by blueforesticarus** [grafana/grafana#50365] [Dashboards, Panels & Visualizations]
When creating graphs, you cannot change the override type.
if you refactor 2 queries into 1, and need to change an override from Query-based to Regex-based, you can't.
You instead need to recreate the whole override.

**Comment by blueforesticarus** [comment]
Manually changing the JSON also doesn't seem to work, perhaps because they are library panels.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
